Extensions/retags.js

```diff
@@ -96,9 +96,9 @@ XKit.extensions.retags = {
 			}
 			if (url) {
 				url = url.split('/');
-				var host = url[2], id = url[4], cache = 'retags_'+id;
-				if (localStorage.getItem(cache) !== null) {
-					XKit.extensions.retags.append($t,cls,$c,JSON.parse(localStorage.getItem(cache)));
+				var host = url[2], id = url[4], cache = 'post_'+id;
+				if (XKit.storage.get('retags', cache, null) !== null) {
+					XKit.extensions.retags.append($t, cls, $c, XKit.storage.get('retags', cache));
```

Comment:
I believe this still needs the JSON.parse call, i.e. `JSON.parse(XKit.storage.get('retags', cache))`